crates/wamr-sys/build.rs

```diff
@@ -162,14 +162,30 @@ fn setup_config(
 }
 
 fn build_wamr_libraries(wamr_root: &PathBuf) {
+    let out_dir = PathBuf::from(env::var("OUT_DIR").unwrap());
+    let vmbuild_path = out_dir.join("vmbuild");
+
     let feature_flags = get_feature_flags();
     let mut cfg = setup_config(wamr_root, feature_flags);
-    let dst = cfg.build_target("iwasm_static").build();
+    let dst = cfg.out_dir(vmbuild_path).build_target("iwasm_static").build();
 
     println!("cargo:rustc-link-search=native={}/build", dst.display());
     println!("cargo:rustc-link-lib=static=vmlib");
 }
 
+fn build_aotclib(wamr_root: &Path) {
```

Comment:
Technically, using the WAMR Rust SDK does not necessitate the use of wamrc. Wamrc is a tool for compiling .wasm files to .aot. May I inquire as to the reason for its inclusion here?